src/main/resources/data/modern_industrialization/recipes/vanilla_recipes/packer/iron_door.json

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "modern_industrialization:packer",
+  "eu": 2,
+  "duration": 100,
+  "item_inputs": [
+    {
+      "item": "modern_industrialization:iron_plate",
+      "amount": 3
+    },
+    {
+      "item": "modern_industrialization:iron_bolt",
```

Comment:
Would remove the bolt as they are not mean to be useful after the steam age ... but it risk to cause conflict

Reply:
I figured that for the more niched items, using early game stuff wouldn't be too big of an issue. Players that aim to automate stuff like iron doors, whether for building or out of completionism, wouldn't mind throwing on an extra machine or two, while the players who are in it purely for progression aren't too affected.

Reply:
That said, made the iron bolt for the trapdoor into an iron ring, to reduce recipe conflicts.